README.md

```diff
@@ -18,37 +18,37 @@
 ## Installation
 
 ```bash
-$ npm install
+$ yarn
 ```
 
 ## Running the app
 
 ```bash
 # development
-$ npm run start
+$ yarn start
 
 # watch mode
-$ npm run start:dev
+$ yarn start:dev
 
 # incremental rebuild (webpack)
-$ npm run webpack
-$ npm run start:hmr
+$ yarn webpack
+$ yarn start:hmr
 
 # production mode
-$ npm run start:prod
+$ yarn start:prod
 ```
 
 ## Test
 
 ```bash
 # unit tests
-$ npm run test
+$ yarn test
 
 # e2e tests
-$ npm run test:e2e
+$ yarn test:e2e
 
 # test coverage
-$ npm run test:cov
+$ yarn test:cov
 ```
 
 ## Environment Configuration
```

Comment:
TODO: add documentation for docker startup

Reply:
added, check if good enough